### PR TITLE
docs: fix custom image for juicefs

### PR DESCRIPTION
**Pull request by zwwhdls**

### CODE REVIEW

File: docs/zh_cn/guide/custom-image.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ cd juicefs
 git checkout ...
 
 # 由于 Dockerfile 在 CSI 驱动的仓库，此处需要自行下载
-curl -O https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/docker/ce.juicefs.Dockerfile
+curl -O https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/docker/dev.juicefs.Dockerfile
 
 # 构建镜像，并上传至私有镜像仓库
 docker build -t registry.example.com/juicefs-csi-mount:ce-latest -f ce.juicefs.Dockerfile .
```
Comment (zxh326) [inline docs/zh_cn/guide/custom-image.md]:
```suggestion
docker build -t registry.example.com/juicefs-csi-mount:ce-latest -f dev.juicefs.Dockerfile .
```

File: docs/en/guide/custom-image.md [hunk 2]
```diff
@@ -129,7 +129,7 @@ cd juicefs
 git checkout ...
 
 # The corresponding Dockerfile resides in the CSI Driver repository
-curl -O https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/docker/ce.juicefs.Dockerfile
+curl -O https://raw.githubusercontent.com/juicedata/juicefs-csi-driver/master/docker/dev.juicefs.Dockerfile
 
 # Build the Docker image, and then push to your private registry
 docker build -t registry.example.com/juicefs-csi-mount:ce-latest -f ce.juicefs.Dockerfile .
```
Comment (zxh326) [inline docs/en/guide/custom-image.md]:
```suggestion
docker build -t registry.example.com/juicefs-csi-mount:ce-latest -f dev.juicefs.Dockerfile .
```

**Review by zxh326 — CHANGES_REQUESTED** [review]
为啥要用 dev呀？

**Comment by zwwhdls** [comment]
@zxh326 因为 ce.juicefs.Dockerfile 里是 git clone juicefs，dev.juicefs.Dockerfile 才是 COPY . .

**Review by zxh326 — APPROVED** [review]

**Pull request merged**